Add option to have a custom slack username and message

BennX opened the pull request:
I added the option to change the slack username and message.

nakiostudio commented on lib/xcov/options.rb:
```diff
@@ -140,6 +140,16 @@ def self.available_options
+          key: :slack_message,
+          description: "The message which is published together with a successful report",
+          optional: true
+        ),
```
I would add a `default_value` argument with `xcov` as value for `slack_username` and `Your *xcov* coverage report` for `slack_message` options. That would simplify the logic within `slack_poster.rb` :)

nakiostudio commented on lib/xcov/slack_poster.rb:
```diff
@@ -8,7 +8,7 @@ def run(report)
       require 'slack-notifier'
       notifier = Slack::Notifier.new(Xcov.config[:slack_url])
-      notifier.username = 'xcov'
+      notifier.username = Xcov.config[:slack_username].to_s.length > 0 ? Xcov.config[:slack_username] : 'xcov'
```
In line with my previous comment this would just be:
```ruby
notifier.username = Xcov.config[:slack_username]
```

nakiostudio commented on lib/xcov/slack_poster.rb:
```diff
@@ -25,8 +25,10 @@ def run(report)
         }
       end
 
+      message = Xcov.config[:slack_message].to_s.length > 0 ? Xcov.config[:slack_message] : "Your *xcov* coverage report"
```
The same but:
```ruby
message = Xcov.config[:slack_message]
````

BennX commented:
@nakiostudio I thought about using the `default_value` but I thought it might be more obvious if I do not do it with the default value. 

Thanks for the feedback. I added it as `default_value` and removed the condition.

BennX commented:
@nakiostudio Cheers 👏
